packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -204,10 +212,11 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
   @override
   Widget build(BuildContext context) {
     final bool closed = !_isExpanded && _controller.isDismissed;
+    final bool shouldRemoveChildren = closed && !widget.maintainState;
     return AnimatedBuilder(
       animation: _controller.view,
       builder: _buildChildren,
-      child: closed ? null : Column(children: widget.children),
+      child: shouldRemoveChildren ? null : Column(children: widget.children),
```

Comment:
Re: https://github.com/flutter/flutter/pull/41886#discussion_r381060438
I would follow the Visibility widget and wrap with the TickerMode and Offstage widgets. I don't know that we need all of the things Visibility would add to it. :)